jdbc/src/main/java/io/kestra/jdbc/runner/JdbcExecutor.java

```diff
@@ -478,8 +480,15 @@ private void sendWorkerTaskResultForWorkerTaskExecution(Execution execution, Wor
 
             maybeWorkerTaskResult.ifPresent(workerTaskResult -> this.workerTaskResultQueue.emit(workerTaskResult));
         } catch (Exception e) {
-            // TODO maybe create a FAILED Worker Task Result instead
             log.error("Unable to create the Worker Task Result", e);
+            // we send a fail worker task result to end the flow
+            this.workerTaskResultQueue.emit(
+                WorkerTaskResult.builder()
+                    .taskRun(taskRun.withState(State.Type.FAILED).withAttempts(
+                        Collections.singletonList(TaskRunAttempt.builder().state(new State().withState(State.Type.FAILED)).build())
+                    ))
```

Comment:
Please merge to previous attempts instead of erasing previous states history

Reply:
Attempts are only handled by the Worker when retrying a task.

Here it's a "fake" WorkerTaskResult that we fail to create from the executable task, there cannot be any existing attempt (and retrying to create the WorkerTaskResult has no meaning).